person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission
+
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
+  end
```

Comment:
- [OPTIONAL] You can simplify the `of_age?` method by using a single comparison `age >= 18` to determine if a student is of age. These simplifications make the code cleaner and more concise while retaining its functionality."
```suggestion
  def of_age?
   age >= 18
  end
```

Reply:
done